sklearn/preprocessing/tests/test_data.py

```diff
@@ -1496,15 +1579,15 @@ def test_quantile_transform_bounds():
     transformer = QuantileTransformer()
     transformer.fit(X)
     assert (transformer.transform([[-10]]) ==
-                 transformer.transform([[np.min(X)]]))
+            transformer.transform([[np.min(X)]]))
     assert (transformer.transform([[10]]) ==
-                 transformer.transform([[np.max(X)]]))
+            transformer.transform([[np.max(X)]]))
     assert (transformer.inverse_transform([[-10]]) ==
-                 transformer.inverse_transform(
-                     [[np.min(transformer.references_)]]))
+            transformer.inverse_transform(
+                [[np.min(transformer.references_)]]))
     assert (transformer.inverse_transform([[10]]) ==
-                 transformer.inverse_transform(
-                     [[np.max(transformer.references_)]]))
+            transformer.inverse_transform(
+                [[np.max(transformer.references_)]]))
```

Comment:
Here again those formatting changes are debatable and unrelated to the topic of the PR.

Reply:
Thanks @ogrisel and @agramfort 
This last changes I did because flake8 was failing locally.
However, I did revert them all now according to your suggestions